### DISCUSSION TITLE
Debugging Uno WASM-enabled web app with Docker enabled

### DISCUSSION THREAD

**Discussion by frogcrush** [unoplatform/uno#17107] [Q&A]
I want to debug my ASP.NET Core server using Docker, however I get the following error message:

![image](https://github.com/unoplatform/uno/assets/6116333/75bca364-d5cd-48b0-944d-6538ff03502f)
``C:\Program Files\dotnet\sdk\8.0.300\Sdks\Microsoft.NET.Sdk.StaticWebAssets\targets\Microsoft.NET.Sdk.StaticWebAssets.References.targets(129,7): error : Manifest file at 'obj\Debug\net8.0-browserwasm\staticwebassets.build.json' not found.
``

I created a new templated project and tried Add > Docker Support and receive the same error. 
This is occurring on Uno SDK versions 5.2.161 and 5.2.139 with .NET 8.0.300

**Comment by jeromelaban** [comment]
Thanks. Can you specify how you're getting to this point, if possible with only what's in the uno template? If not, can you provide a repro sample?

**Comment by frogcrush** [comment]
Sure. I create a new Uno platform WInUI app, and select iOS/Windows/Android/Webassembly. Then I select the Server hosting project. 

![image](https://github.com/unoplatform/uno/assets/6116333/98ef152c-f03d-4ad8-a49c-28c8512040bd)

Add Docker support to the web project:

![image](https://github.com/unoplatform/uno/assets/6116333/05354e0b-dadc-4c01-a252-1c53dfb33da4)

Set the server as the startup project and try to debug. 
![image](https://github.com/unoplatform/uno/assets/6116333/19679d34-dc01-4ba4-a3ec-58bf4f909bba)

Repro: https://github.com/frogcrush/UnoDockerIssue

**Reply by jeromelaban** [reply]
Thanks. To be sure, when you're saying the "Web project" which one is it? Is it the `.Server` project? The screenshot does not show it.

**Reply by frogcrush** [reply]
Yes, the server project.

**Reply by jeromelaban** [reply]
Thanks. I've tested this again with our latest uno dev builds (5.3.0-dev.1903) and it's working properly. Could you test this on your side and let me know if it helps?

**Reply by frogcrush** [reply]
Thanks! I tested it with that build set in my global.json file but I still receive the error:

```
Build started at 12:18 AM...
Restored G:\Repos\UnoApp1\UnoApp1.DataContracts\UnoApp1.DataContracts.csproj (in 0.7 ms).
Restored G:\Repos\UnoApp1\UnoApp1.Server\UnoApp1.Server.csproj (in 28 ms).
Restored G:\Repos\UnoApp1\UnoApp1\UnoApp1.csproj (in 854 ms).
1>------ Build started: Project: UnoApp1.DataContracts, Configuration: Debug Any CPU ------
1>Skipping analyzers to speed up the build. You can execute 'Build' or 'Rebuild' command to run analyzers.
1>UnoApp1.DataContracts -> G:\Repos\UnoApp1\UnoApp1.DataContracts\bin\Debug\net8.0\UnoApp1.DataContracts.dll
2>------ Build started: Project: UnoApp1, Configuration: Debug Any CPU ------
2>Skipping analyzers to speed up the build. You can execute 'Build' or 'Rebuild' command to run analyzers.
2>UnoApp1 -> G:\Repos\UnoApp1\UnoApp1\bin\Debug\net8.0-windows10.0.19041\UnoApp1.dll
3>------ Build started: Project: UnoApp1.Server, Configuration: Debug Any CPU ------
3>Skipping analyzers to speed up the build. You can execute 'Build' or 'Rebuild' command to run analyzers.
3>C:\Program Files\dotnet\sdk\8.0.300\Sdks\Microsoft.NET.Sdk.StaticWebAssets\targets\Microsoft.NET.Sdk.StaticWebAssets.References.targets(129,7): error : Manifest file at 'obj\Debug\net8.0-browserwasm\staticwebassets.build.json' not found.
3>Done building project "UnoApp1.csproj" -- FAILED.
========== Build: 2 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
========== Build completed at 12:18 AM and took 05.502 seconds ==========
========== Deploy: 0 succeeded, 0 failed, 0 skipped ==========
========== Deploy completed at 12:18 AM and took 05.502 seconds ==========
```

I did a clean, and also tried deleting the bin and obj folders but same result. 

I also tried updating Uno.Wasm.Bootstrap.Server to 9.0.0-dev.86 and the same error occurred.

**Reply by jeromelaban** [reply]
Thanks for the update. Could you provide [binlogs for your build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?

**Reply by jeromelaban** [reply]
@Darsh0307 could you try reproducing this issue?

**Reply by frogcrush** [reply]
> Thanks for the update. Could you provide [binlogs for your build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?

[UnoApp1.Server_Debug_AnyCPU_Build_2024-06-17T10_06_45.9915859-04_00.zip](https://github.com/user-attachments/files/15872335/UnoApp1.Server_Debug_AnyCPU_Build_2024-06-17T10_06_45.9915859-04_00.zip)

**Reply by Darsh0307** [reply]
> @Darsh0307 could you try reproducing this issue?

@jeromelaban 
I Its completely working fine when i am trying to run

Uno templates:- 5.3.2

**Reply by frogcrush** [reply]
I have it working right now, and I think what affected it was building the WASM project first once, and without doing a clean running the docker build from VS. It also seems to require 5.3.0-dev.1903. I want to test further by doing a full clean/deleting /obj
